_plugins/style_guide_assets.rb

```diff
@@ -7,7 +7,9 @@
 
 def copy_assets(origin, destination)
   origin_glob = File.join(origin, '**/*')
-  origin_files = Dir.glob(origin_glob).reject { |f| File.directory?(f) }
+  origin_files = Dir.glob(origin_glob).
+    reject { |f| File.directory?(f) }.
+    reject { |f| f.include?('/favicons/') }
```

Comment:
If we're excluding it from being copied over, does it also still need to be present in the Jekyll `exclude` config? I would think we'd just need one or the other?

Reply:
Yes, exactly the linter caught extra `ico` files. I think that this plugin basically blindly copies files, so that `excludes` line got ignored, I can remove it so it doesn’t look like it works